src/kademlia/item.cpp

```diff
@@ -140,10 +140,12 @@ item::item(entry v)
 {}
 
 item::item(bdecode_node const& v)
-	: m_value(std::move(v))
-	, m_seq(0)
+	: m_seq(0)
 	, m_mutable(false)
-{}
+{
+	// TODO: implement ctor for entry from bdecode_node?
+	m_value = v;
```

Comment:
is ``m_value`` not a ``bdecode_node``? I'm surprised std::move() worked from a ``const&`` previously